### PR TITLE
integrations section started

### PR DESCRIPTION
**Pull request by paulo-valim**
@josevalim I started the section. Before I start the logos and texts, I want to clear the design with you; that's why it is not ready yet. I also made it responsive; please check it out on other device screens. =)

### CODE REVIEW

**Comment by josevalim** [comment]
1. We can definitely decrease the size of each integration.

2. On small screens, we still want to show _all_ integrations. For md, perhaps we show 2->3->2->3 and for sm, we show 2->2->2->2->2.

**Comment by josevalim** [comment]
@paulo-valim, this is a better indicator of the size:

image

I used h-40 for the height of the grid. Inside the cell, I used:

```html
        <img
        class="h-[60px]"
        src="/images/Integrations/elixir_logo.svg"
        alt="Livebook Aplication Screenshot"
        />
        <span class="text-gray-700 font-medium text-base text-lg mt-2">Elixir</span>
        <span class="text-gray-500 font-medium md:font-semibold text-xs tracking-wider sm:tracking-widest">LANGUAGE</span>
```

I saw that for large screens you are showing 5 columns, but we should continue showing 4. You should use `max-w-sm ` around the grid so it doesn't grow too much.

File: src/index.html [hunk 1]
```diff
@@ -262,6 +262,125 @@ <h3 class="text-xl text-gray-50 font-semibold">
   </div>
 </section>
 
+<!-- Integrations -->
+<section id="integrations">
+  <div class="m-auto max-w-7xl px-4 py-12 md:pt-20 flex flex-col items-center">
```
Comment (josevalim) [inline src/index.html:267]: max-x-7xl is unused:

```suggestion
  <div class="m-auto px-4 py-12 md:pt-20 flex flex-col items-center">
```

File: src/index.html [hunk 2]
```diff
@@ -262,6 +262,125 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+    <h2 class="text-gray-900 text-2xl md:text-3xl font-semibold text-center">
+      Integrations
+    </h2>
+    <span class="mt-3 md:max-w-sm text-gray-700 font-medium text-center">
+      It's easy to get start with Livebook, whether it be on the cloud or your local machine.
+    </span>
+    <a class="mt-8 max-w-7xl button button-outlined-gray whitespace-nowrap" href="#integrations">Browse Integrations</a>
```
Comment (josevalim) [inline src/index.html:274]: `max-w-7xl` is unused:

```suggestion
    <a class="mt-8 button button-outlined-gray whitespace-nowrap" href="#integrations">Browse Integrations</a>
```

File: src/index.html [hunk 3]
```diff
@@ -48,7 +48,7 @@ <h2 class="max-w-lg text-3xl font-light xl:text-left text-gray-900">
     <img
       class="xl:max-w-[70%] xl:ml-20 rounded-md custom-shadow-1"
       src="/images/welcome_to_livebook_screenshot.png"
-      alt="Livebook Aplication Screenshot"
+      alt="Logo"
```
Comment (josevalim) [inline src/index.html:51]:
```suggestion
      alt="Livebook Application Screenshot"
```

File: src/index.html [hunk 4]
```diff
@@ -262,11 +262,190 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+<!-- Integrations -->
+<section id="integrations">
+  <div class="m-auto px-4 py-12 md:pt-20 flex flex-col items-center">
+    <h2 class="text-gray-900 text-2xl md:text-3xl font-semibold text-center">
+      Integrations
+    </h2>
+    <span class="mt-3 md:max-w-sm text-gray-700 font-medium text-center">
+      It's easy to get start with Livebook, whether it be on the cloud or your local machine.
+    </span>
+    <a class="mt-8 button button-outlined-gray whitespace-nowrap" href="#integrations">Browse Integrations</a>
```
Comment (josevalim) [inline src/index.html:274]: We will have this button commented out for now:

```suggestion
    <!-- <a class="mt-8 button button-outlined-gray whitespace-nowrap" href="#integrations">Browse Integrations</a> -->
```

:)

File: src/index.html [hunk 5]
```diff
@@ -262,11 +262,190 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+          alt="MapLibre Logo"
+          />
+          <span class="text-gray-700 font-medium text-base lg:text-lg mt-2">MapLibre</span>
+          <span class="text-gray-500 font-medium md:font-semibold text-xs tracking-wider sm:tracking-widest">VISUALIZATION</span>
+        </div>
+        <!-- Mermaid -->
+        <div class="hidden md:flex md:flex-col border-y border-r border-gray-300 justify-center items-center">
+          <img
+          class="h-[40px] lg:h-[60px]"
+          src="/images/Integrations/Mermaid_logo.png"
```
Comment (josevalim) [inline src/index.html:387]: You can find their actual logo on Twitter: https://twitter.com/mermaidjs_

File: src/index.html [hunk 6]
```diff
@@ -262,11 +262,190 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+          alt="Logo"
+          />
+          <span class="text-gray-700 font-medium text-base lg:text-lg mt-2">Elixir</span>
+          <span class="text-gray-500 font-medium md:font-semibold text-xs tracking-wider sm:tracking-widest">LANGUAGE</span>
+        </div>
+        <!-- SQL -->
+        <div class="flex flex-col border-y md:border-x border-gray-300 justify-center items-center">
+          <img
+          class="h-[40px] lg:h-[60px]"
+          src="/images/Integrations/SQL_logo.png"
```
Comment (josevalim) [inline src/index.html:299]: This logo actually belongs to Microsoft Azure. My suggestion is to use one of the Remix icons for database:  https://remixicon.com/

File: src/index.html [hunk 7]
```diff
@@ -262,11 +262,190 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+        </div>
+        <div class="hidden md:flex border-y border-gray-300">
+        </div>
+      </div>
+      <div class="grid grid-cols-it-2 md:grid-cols-it-md-4 lg:grid-cols-it-lg-4 xl:grid-cols-it-xl-4 h-28 lg:h-36 xl:h-40" >
+        <!-- PostgreSQL -->
+        <div class="hidden md:flex md:flex-col border-x border-gray-300 justify-center items-center">
+          <img
+          class="h-[40px] lg:h-[60px]"
+          src="/images/Integrations/PostgreSQL_logo.svg"
```
Comment (josevalim) [inline src/index.html:323]: Missing logo :)

File: src/index.html [hunk 8]
```diff
@@ -262,11 +262,190 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+    class="hidden md:block absolute left-0 h-[100%] w-[30%] bg-gradient-to-l from-transparent to-white/100"></div>
+    <div class="m-auto md:flex md:flex-col justify-center items-center">
+      <div class="grid grid-cols-it-2 md:grid-cols-it-md-5 lg:grid-cols-it-lg-5 xl:grid-cols-it-xl-5 h-28 lg:h-36 xl:h-40">
+        <div class="hidden md:flex border-y border-gray-300">
+        </div>
+        <!-- Elixir -->
+        <div class="flex flex-col border-y border-x md:border-l md:border-x-0 border-gray-300 justify-center items-center">
+          <img
+          class="h-[40px] lg:h-[60px]"
+          src="/images/Integrations/Elixir_logo.svg"
```
Comment (josevalim) [inline src/index.html:289]: Missing logo :)

File: src/index.html [hunk 9]
```diff
@@ -262,11 +262,190 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+          <span class="text-gray-500 font-medium md:font-semibold text-xs tracking-wider sm:tracking-widest">DATABASE</span>
+        </div>
+        <!-- SQLite -->
+        <div class="flex flex-col border-x md:border-x-0  border-gray-300 justify-center items-center">
+          <img
+          class="h-[40px] lg:h-[60px]"
+          src="/images/Integrations/SQL-lite_logo.svg"
+          alt="SQLite Logo"
+          />
+          <span class="text-gray-700 font-medium text-base lg:text-lg mt-2">SQL lite</span>
```
Comment (josevalim) [inline src/index.html:336]:
```suggestion
          <span class="text-gray-700 font-medium text-base lg:text-lg mt-2">SQLite</span>
```

File: src/index.html [hunk 10]
```diff
@@ -262,11 +262,190 @@ <h3 class="text-xl text-gray-50 font-semibold">
...
+    class="hidden md:block absolute left-0 h-[100%] w-[27%] bg-gradient-to-l from-transparent via-white/75 to-white/100"></div>
+    <div class="m-auto md:flex md:flex-col justify-center items-center">
+      <div class="grid grid-cols-it-2 md:grid-cols-it-md-5 lg:grid-cols-it-lg-5 xl:grid-cols-it-xl-5 h-28 lg:h-36 xl:h-40">
+        <div class="hidden md:flex border-y border-gray-300">
+        </div>
+        <!-- Elixir -->
+        <div class="flex flex-col border-y border-x md:border-l md:border-x-0 border-gray-300 justify-center items-center">
+          <img
+          class="h-[40px] lg:h-[60px]"
+          src="/images/Integrations/Elixir_logo.svg"
```
Comment (josevalim) [inline src/index.html:289]:
```suggestion
          src="/images/Integrations/elixir_logo.svg"
```

**Comment by hugobarauna** [comment]
Can we add "Integrations" to the menu as well?

Like this:

CleanShot 2022-12-01 at 18 22 10@2x

When the visitor clicks that menu item, the site scrolls to the integrations section.

That way, we can bring awareness that we have integrations without requiring the visitor to scroll.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**